plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseTypeMapping.java

```diff
@@ -27,11 +25,6 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         clickhouseServer = closeAfterClass(new TestingClickHouseServer());
-        return createClickHouseQueryRunner(clickhouseServer, ImmutableMap.of(),
-                ImmutableMap.<String, String>builder()
-                        .put("metadata.cache-ttl", "10m")
-                        .put("metadata.cache-missing", "true")
-                        .buildOrThrow(),
-                ImmutableList.of());
+        return createClickHouseQueryRunner(clickhouseServer);
```

Comment:
any specific reason to remove metadata cache from type mapping tests?

Reply:
Here's elapsed time comparison in my laptop. No improvement by metadata cache.
* with cache: 1m 36s
* without cache: 1m 33s